AEPCore/Sources/migration/IDParser.swift

```diff
@@ -46,7 +48,7 @@ struct IDParser: IDParsing {
                 continue
             }
 
-            let IdDict = ["origin": currentCustomerIdOrigin, "type": idValueComponents[0], "id": idValueComponents[1], "authenticationState": Int(idValueComponents[2]) ?? 0] as [String: Any]
+            let IdDict = ["id_origin": originValue, "id_type": idValueComponents[0], "id": idValueComponents[1], "authentication_state": Int(idValueComponents[2]) ?? 0] as [String: Any]
```

Comment:
Looks like these didn't get updated either when the keys were changed.

I think I'd prefer to see them something like: `[CustomIdentity.CodingKeys.origin :originValue]`

Reply:
Actually, I guess that may not be possible as CustomIdentity is in AEPIdentity target.

Reply:
Yes. It would cause a circular dependency between AEPCore and AEPIdentity, unless we moved CustomIdentity to AEPCore. I don't feel it is necessary to do that for this case, however.